cmd/horologium/main.go

```diff
@@ -199,15 +203,23 @@ func sync(prowJobClient ctrlruntimeclient.Client, cfg *config.Config, cr cronCli
 			}
 			continue
 		}
-		if !shouldTrigger {
+		if !shouldTrigger && p.RetriggerFailed == nil {
```

Comment:
This will remove the logging even when no retry is in order (last execution was a success and we wait for next vanilla trigger)

Reply:
The intention was not to use this comment which is a debug level if the feature is enabled but use for old jobs. Not sure what to do with it to avoid complicated if.